Mathlib/Tactic/Ring/Basic.lean

```diff
@@ -3,7 +3,9 @@ Copyright (c) 2018 Mario Carneiro. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro, Aurélien Saue, Anne Baanen
 -/
+import Mathlib.Algebra.CharP.Defs
 import Mathlib.Algebra.Order.Ring.Rat
+import Mathlib.Tactic.NormNum.DivMod
```

Comment:
@Vierkantor can you quickly confirm this has been resolved correctly now when I merged `master`?